prompt.py

```diff
@@ -250,47 +252,75 @@ def do_close_wallet(self):
     def do_import(self, arguments):
         item = get_arg(arguments)
 
-        if item:
```

Comment:
It looks like much changes here, but I only put the `if not item` test on top and then could de-indent the section. perhaps easier see by looking at the current code here: https://github.com/metachris/neo-python/blob/nep2-prompt/prompt.py#L255